content/en/lotus/manage/chain-management.md

```diff
@@ -36,13 +36,13 @@ These lightweight state snapshots **do not contain any message receipts**. To ge
     a. For **mainnet**, command always contains the latest snapshot available for mainnet:
 
     ```shell
-    curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car | perl -ne '/x-amz-website-redirect-location:\s(.+)\.car/ && print "$1.sha256sum\n$1.car"' | xargs wget
+    curl -sI https://snapshots.mainnet.filops.net/minimal/latest | perl -ne '/x-amz-website-redirect-location:\s(.+)\.car/ && print "$1.sha256sum\n$1.car"' | xargs wget
```

Comment:
This will not work: the filops redirect does not have an `x-amz-website-redirect-location` header. In general the entire paragraph about verifying checksums is bogus: car files are self-validating. This needs to be removed, @rjan90 has more context.

Reply:
Whoops! Good thing you where lurking here as well - I will make some more changes. Thanks!